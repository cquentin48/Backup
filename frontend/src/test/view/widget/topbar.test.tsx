import React from "react";

import { render } from "@testing-library/react"
import TopBar from "../../../main/app/view/pages/computer/topbar";

import '@testing-library/jest-dom'

describe("Topbar unit test suite", () => {
    test("Successful render", async () => {
        // Acts
        const { asFragment } = render(
            <TopBar />
        )

        // Assert
        expect(asFragment()).toMatchSnapshot()
    })
})
