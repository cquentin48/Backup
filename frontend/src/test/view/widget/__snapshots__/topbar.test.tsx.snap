// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topbar unit test suite Successful render 1`] = `
<DocumentFragment>
  <div
    id="TopBar"
  >
    Top bar here!
  </div>
</DocumentFragment>
`;
